system-parachains/bridge-hubs/bridge-hub-polkadot/src/lib.rs

```diff
@@ -500,22 +499,17 @@ impl pallet_utility::Config for Runtime {
 	type WeightInfo = weights::pallet_utility::WeightInfo<Runtime>;
 }
 
-// Ethereum Bridge
-parameter_types! {
-	// The gateway address is set by governance.
-	pub storage EthereumGatewayAddress: H160 = H160::zero();
-}
-
+// Ethereum Bridge pallets
 impl snowbridge_pallet_inbound_queue::Config for Runtime {
```

Comment:
@claravanstaden I meant everything and pallets setup also :), everything related to the Eth, move to the `bridge_to_ethereum_config.rs` please :)